fairlearn/metrics/_group_metric_set.py

```diff
@@ -44,6 +45,7 @@
 # Issue 269 is about unifying the two sets
 ACCURACY_SCORE = "accuracy_score"
 BALANCED_ROOT_MEAN_SQUARED_ERROR = "balanced_root_mean_squared_error"
+COUNT = "count"
```

Comment:
It's harmless to add this, but also I don't think it'll be used. This routine is only for the dashboard, and will be removed from Fairlearn in the near future.

Reply:
And when it's moved won't this be useful to have it included? Otherwise you'll need to manually add it?